raster.py

```diff
@@ -535,20 +553,43 @@ def create_bounds_from_res(res_x: int, res_y: int, base_bounds: Bounds):
   return new_bounds
 
 def generate_isoscapes_from_variational_model(
-    output_geotiff_id: str,
     model: model.Model,
     required_geotiffs: List[str],
     res_x: int, 
-    res_y: int):
+    res_y: int,
+    output_geotiff: str,
+    amazon_only: bool=False):
```

Comment:
i think a separate method to trim an isoscape is more generally useful because you can trim existing isoscapes rather than have to re-run them thru the ml colab.
otherwise, how will you run assignR validation on already validated brazil isoscapes.

that said, i dont think its worth blocking the PR for.